io.github.jakubmelka.pdf4qt.yaml

```diff
@@ -0,0 +1,27 @@
+app-id: io.github.jakubmelka.pdf4qt
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+command: pdf4qtbin.sh
```

Comment:
you are missing finish-args to at least give the permission to display something